src/workspace/src/browser.ts

```diff
@@ -1,11 +1,12 @@
 import {
-  BrowserOverrideCallback,
   getCurrentSync,
   GlobalContextMenuOptionType,
   OpenGlobalContextMenuPayload,
-  Page
-} from '@openfin/workspace-platform'
-import { getUserToSwitch, switchUser } from './user'
+  Page,
```

Comment:
minor niggle this, but let's do a one-off pass of prettying on the codebase, we have had a series of PRs against /workspace now and it's hard to see the _real_ diffs (I am as guilty of this as anyone, btw, not a criticism of this PR alone)